src/Task/GherkinLint.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Formatter\ProcessFormatterInterface;
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Runner\TaskResultInterface;
+use GrumPHP\Task\Config\ConfigOptionsResolver;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @extends AbstractExternalTask<ProcessFormatterInterface>
+ */
+class GherkinLint extends AbstractExternalTask
+{
+    public static function getConfigurableOptions(): ConfigOptionsResolver
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults([
+            'directory' => 'features',
+            'config' => null,
+        ]);
+
+        $resolver->addAllowedTypes('directory', ['string']);
+        $resolver->addAllowedTypes('config', ['null', 'string']);
+        $resolver->addAllowedValues('config', [null, 'text']);
```

Comment:
shouldn't config be any string? Now you are limiting it to the literal `text` here ?

On second notice: there doesn't seem a way to specify the configuration file. It assumes a hardcoded path instead:

* https://github.com/dantleech/gherkin-lint-php?tab=readme-ov-file#configuration
* https://github.com/dantleech/gherkin-lint-php/blob/f3277c4945261d26a3098b5e003b51d815eaed4b/bin/gherkinlint#L20-L27
* https://github.com/dantleech/gherkin-lint-php/blob/f3277c4945261d26a3098b5e003b51d815eaed4b/src/Command/LintCommand.php#L27-L32

So I'dd say its better to remove the config option alltogether.